_stbt/control.py

```diff
@@ -9,6 +9,8 @@
 import time
 from contextlib import contextmanager
 
+import requests
```

Comment:
I think this probably deserves to be imported in press (or init).  That way it becomes a optional dependency that is only required if you use the roku remote control.